### PR TITLE
Change test_it_can_create_games_from_csv to not use mock/stubs

### PR DESCRIPTION
**Pull request by javier-aguilar**
I have updated the GameCollectionTest to not use mock and stubs since we are generating game objects.

### CODE REVIEW

File: test/game_collection_test.rb [hunk 1]
```diff
@@ -17,13 +17,33 @@ def test_it_exists
...
+    assert_instance_of Game, game2
+    assert_equal 3, game2.away_goals
+    assert_equal 16, game2.away_team_id
+    assert_equal "4/18/14", game2.date_time
+    assert_equal 2, game2.home_goals
+    assert_equal 19, game2.home_team_id
+    assert_equal 0, game2.id
+    assert_equal "20132014", game2.season
+    assert_equal "Postseason", game2.type
   end
```
Comment (Shock253) [inline test/game_collection_test.rb:47]: Looks good, my testing for TeamCollection looks very similar so far

**Review by Shock253 — APPROVED** [review]
Looks good, testing is very similar to the testing I have for TeamCollection

I'm itching to start refactoring our code to implement inheritance

File: test/game_collection_test.rb (hunk 1 above)
Comment (mlynch5187) [reply test/game_collection_test.rb:47]: Good work on game_collection_test, I was just doing something similar.

**Review by mlynch5187 — APPROVED** [review]
I approve of merging this, it is a logical update to game_collection_test

**Pull request merged**